rise-logger-utils.html

```diff
@@ -0,0 +1,104 @@
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`rise-logger-utils` is a web component that is a utilities helper for `rise-logger`.
+
+-->
+
+<script>
+  (function() {
+    /* global Polymer */
+    /* jshint newcap: false */
+
+    "use strict";
+
+    var USAGE_DEV = "dev",
+      USAGE_STANDALONE = "standalone",
+      USAGE_WIDGET = "widget";
+
+    Polymer({
+      is: "rise-logger-utils",
+
+      hostAttributes: {
+        hidden: true
+      },
+
+      _getSuffix: function() {
+        var date = new Date(),
+          year = date.getUTCFullYear(),
+          month = date.getUTCMonth() + 1,
+          day = date.getUTCDate();
+
+        if (month < 10) {
+          month = "0" + month;
+        }
+
+        if (day < 10) {
+          day = "0" + day;
+        }
+
+        return year + month + day;
+      },
+
+      _getUsageType: function(href) {
+        var a = document.createElement("a"),
+          type = USAGE_STANDALONE;
+
+        a.href = href;
+
+        if (a.hostname === "localhost") {
+          type = USAGE_DEV;
+        }
+        else if (a.hostname === "s3.amazonaws.com" && a.pathname.indexOf("widget-") > -1) {
```

Comment:
@donnapep I think they're good ideas you mention but also agree it needs its own card to figure out best approach since we're trying to solve a problem that may never arise. What I can do for now though is check against our exact widget names (eg. widget-google-spreadsheet) to increase the specificity and lower the chances that usage type gets wrong association. I think that should suffice for now. Sound good?